frontend/src/relay.ts

```diff
@@ -0,0 +1,11 @@
+import { Environment, Store, RecordSource, Network } from "relay-runtime";
+
+export const environment = new Environment({
+    store: new Store(new RecordSource()),
+    network: Network.create(({ text: query }, variables) =>
+        fetch("/graphql", {
+            method: "POST",
+            headers: { "Content-Type": "application/json" },
+            body: JSON.stringify({ query, variables }),
+        }).then(response => response.json())),
```

Comment:
MPHF

```suggestion
    network: Network.create(({ text: query }, variables) =>
        fetch("/graphql", {
            method: "POST",
            headers: { "Content-Type": "application/json" },
            body: JSON.stringify({ query, variables }),
        }).then(response => response.json())
    ),
```

Reply:
No. Then it has to be

```suggestion
    network: Network.create(
        ({ text: query }, variables) =>
            fetch("/graphql", {
                method: "POST",
                headers: { "Content-Type": "application/json" },
                body: JSON.stringify({ query, variables }),
            }).then(response => response.json())
    ),

```

or

```suggestion
    network: Network.create(
        ({ text: query }, variables) => fetch("/graphql", {
            method: "POST",
            headers: { "Content-Type": "application/json" },
            body: JSON.stringify({ query, variables }),
        }).then(response => response.json())
    ),
```

Or something like this. I refuse to break before the closing `)`, but not after the corresponding `(`. x)

Reply:
Yeah, one of those please :P